docs/2.8.0/docs/kubernetes/04-deploy-daml-enterprise/01-helm-deployment/03-domain-deployment.rst

```diff
@@ -0,0 +1,148 @@
+.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Install ``canton-domain`` Chart
+###############################
+
+Objectives
+**********
+
+* Customize Helm chart parameters
+* Install Helm chart
+* Verify the status of the deployment
+
+Steps
+*****
+
+1. Inspect the Helm value files generated by Terraform
+======================================================
+
+.. note::
+   The Terraform scripts were used to parameterize the Helm values. If standalone Helm deployment is done without Terraform, the value file has to be customized manually. The value file can be found under ``domain.yaml``.
```

Comment:
I copied the domain.yaml file and put it in the docs repo. Need to figure out how to link to it.